packages/flutter/test/widgets/interactive_viewer_test.dart

```diff
@@ -366,6 +366,123 @@ void main() {
       expect(translation.y, 0.0);
     });
 
+    testWidgets('panningDirection allows panning in one direction only for diagonal gesture', (WidgetTester tester) async {
```

Comment:
I think you forgot to update this: "panningDirection" => "PanAxis.horizontal"

Also instead of saying "one direction only" maybe say "the horizontal direction only".

Reply:
Same for the other tests below.